en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/observability/distributed-tracing.md

```diff
@@ -0,0 +1,124 @@
+# Distributed Tracing
+
+Performance issues, errors, and exceptions are unfortunate events that may occur in a production environment. In order to identify such an unfortunate event, observing the production environment is essential. WSO2 Choreo Connect provides the ability to observe how the Gateway is handling requests via OpenTelemetry based distributed tracing implementation. By connecting WSO2 Choreo Connect to one of the supported distributed tracing systems, users are able to easily debug and identify production issues.
+
+Supported distributed tracing systems,
+
+- Jaeger
+- Zipkin
+- Azure Application Insights
+
+## Configure Distributed Tracing
+
+!!! note
+    If you are trying out tracing capabilities of WSO2 Choreo Connect and do not have an actual deployment of Jaeger or Zipkin, you can start Choreo Connect deployment with Jaeger/Zipkin by adding one of the below containers to product docker-compose file.
+
+    ```yaml tab="Jaeger"
+    jaeger:
+      image: jaegertracing/all-in-one:1.27
+      environment:
+        - COLLECTOR_ZIPKIN_HOST_PORT=9411
+      ports:
+        - "5775:5775"
+        - "16686:16686"
+        - "14268:14268"`
+    ```
+
+    ```yaml tab="Zipkin"
+    zipkin:
+      image: openzipkin/zipkin
+      container_name: zipkin
+      ports:
+        - "9411:9411"
+    ```
+
+### Jaeger
+
+When using Jaeger for tracing, you need to use zipkin format to publish span from Choreo Connect. Therefor tracer type is configured as zipkin.
+Follow these steps to configure WSO2 Choreo Connect with Jaeger,
+
+1. Set following environment variable to `jaeger` container/pod.
+    `COLLECTOR_ZIPKIN_HOST_PORT=9411`
+1. Add the following configuration to the `config.toml` file.  
+    ```toml
+    [tracing]
+      enabled = true
+      type = "zipkin"
+      [tracing.configProperties]
+        host = "jaeger"
+        port = "9411"
+        endpoint = "/api/v2/spans"
+        instrumentationName = "CHOREO-CONNECT"
+        maximumTracesPerSecond = "2"
```

Comment:
We need to define the maxPathLength as well. Otherwise Router tracing does not work

Reply:
added with bd496f4